### DISCUSSION TITLE
How to configure downloading with skipping sponsorblock segments?

### DISCUSSION THREAD

**Discussion by ghost** [JunkFood02/Seal#253] [Q&A] [answered]
I want to skip Sponsor, Interaction Reminder (Subscribe), Music: Non-Music Section, Intermission/Intro Animation, Endcards/Credits. I tried to write it like "Sponsor", "Endcards/Credits", but it doesn't work that way. How to write it correctly?

**Comment by Marco-9456** [comment, marked as answer]
> I want to skip Sponsor, Interaction Reminder (Subscribe), Music: Non-Music Section, Intermission/Intro Animation, Endcards/Credits. I tried to write it like "Sponsor", "Endcards/Credits", but it doesn't work that way. How to write it correctly?

See https://github.com/yt-dlp/yt-dlp#sponsorblock-options

**Reply by ghost** [reply]
There's a place to write into app. Should I write --sponsorblock-mark CATS  or something different?

**Reply by Marco-9456** [reply]
> There's a place to write into app. Should I write --sponsorblock-mark CATS or something different?

Can you please provide a link for testing

**Reply by ghost** [reply]
https://user-images.githubusercontent.com/85103965/192885705-d8b2de51-c9c5-44e5-9d1b-347464ed8df3.mp4

You mean where's the place to write?

**Reply by Marco-9456** [reply]
> video_2022-09-28_23-47-40.mp4 
> You mean where's the place to write?

Forget about my above comment,

You just need to right the categories of what you want to be deleted from the video.
These are the categories that i know: `sponsor, intro, outro, selfpromo, preview, filler, interaction, music_offtopic,`
If you want to delete only the preview then right just: `preview` or if you want to delete the preview and selfpromo for example then you need to separate them with a , like this:  `preview, selfpromo`

![Screenshot_20220928_230310](https://user-images.githubusercontent.com/41828058/192888401-d4893284-aaf7-4b75-8144-9b498662b7eb.jpg)

**Reply by ghost** [reply]
**Thank you, it helped me!**

**Reply by Marco-9456** [reply]
> **Thank you, it helped me!**

Don't mention it :)
